rootfs/usr/local/include/toolbox/kops/Makefile.defaults

```diff
@@ -10,13 +9,8 @@ export KOPS_STATE_STORE ?= s3://$(CLUSTER_STATE_BUCKET)
 
 export KOPS_CLOUD ?= aws
 export KOPS_ZONES ?= $(AWS_REGION)a
-
-export KOPS_MASTER_SIZE ?= t2.medium
 export KOPS_MASTER_ZONES ?= $(KOPS_ZONES)
 
-export KOPS_NODE_SIZE ?= t2.medium
```

Comment:
The vars have been moved to Dockerfile, not from geodesic repo, but to the cluster-related repos.
We probably don't want anything cluster-specific (and user-configurable) to be in the generic geodesic image.

https://github.com/cloudposse/cluster-4.cloudposse.com/blob/master/Dockerfile
https://github.com/gadgetreview/aws.gadgetreview.com/blob/fix-env-vars/Dockerfile